### PR TITLE
debian: replace SysV rbdmap with systemd service

### PR DESCRIPTION
**Pull request by ktdreyer**
Stop shipping `/etc/init.d/rbdmap` in the Debian packages. Ship the `rbdmap.service` systemd unit file instead.

The corresponding change has already been made for RPMs, in 9224ac2ad25f7d017916f58b642c0ea25305c3e5.

For Upstart-based systems (eg Ubuntu Trusty), the Debian packages already contain `rbdmap.conf`.

(This gets us a tiny bit closer to being able to remove the rbdmap SysV script from our tree entirely.)

### CODE REVIEW

**Comment by ktdreyer** [comment]
cc @javacruft in case he wants to review

File: debian/rules [hunk 1]
```diff
@@ -39,7 +39,6 @@ override_dh_auto_install:
 	install -D -m 644 udev/95-ceph-osd.rules $(DESTDIR)/lib/udev/rules.d/95-ceph-osd.rules
 	install -D -m 644 udev/60-ceph-by-parttypeuuid.rules $(DESTDIR)/lib/udev/rules.d/60-ceph-by-parttypeuuid.rules
 	install -D -m 644 src/etc-rbdmap $(DESTDIR)/etc/ceph/rbdmap
-	install -D -m 755 src/init-rbdmap $(DESTDIR)/etc/init.d/rbdmap
```
Comment (tchaikov) [inline debian/rules]: are we dropping sysv support completely? if this is the case, shall we remove `/etc/init.d/ceph` as well?

Comment (ktdreyer) [inline debian/rules]: we could do that in a separate PR?

**Comment by tchaikov** [comment]
lgtm,

File: debian/rules [hunk 2]
```diff
@@ -85,6 +84,7 @@ override_dh_installinit:
 	install -m0644 systemd/ceph-create-keys@.service debian/ceph-base/lib/systemd/system
 	install -m0644 systemd/ceph-osd@.service debian/ceph-osd/lib/systemd/system
 	install -m0644 systemd/ceph-disk@.service debian/ceph-osd/lib/systemd/system
+	install -m0644 systemd/rbdmap.service debian/ceph/lib/systemd/system
```
Comment (tchaikov) [inline debian/rules]: @ktdreyer maybe we should install `rbdmap.service` into `debian/ceph-common/lib/systemd/system` instead? see http://gitbuilder.sepia.ceph.com/gitbuilder-ceph-deb-trusty-amd64-basic/log.cgi?log=d226051ec096b331497c3c90ff6767670302d3a9

```
install -m0644 systemd/rbdmap.service debian/ceph/lib/systemd/system
install: cannot create regular file ‘debian/ceph/lib/systemd/system’: No such file or directory
make[1]: *** [override_dh_installinit] Error 1
make[1]: Leaving directory `/srv/autobuild-ceph/gitbuilder.git/build/out~/ceph-11.0.0-1613-gd226051'
make: *** [binary] Error 2
error: dpkg-buildpackage: fakeroot debian/rules binary gave error exit status 2
```

Comment (ktdreyer) [inline debian/rules]: Great catch, thank you! I've re-pushed my `wip-debian-rbdmap-service` branch.

**Comment by tchaikov** [comment]
tested at http://pulpito.ceph.com/kchai-2016-08-20_18:41:17-rados-wip-kefu-testing---basic-mira/

failed ones are tracked by 

* http://tracker.ceph.com/issues/17087
* http://tracker.ceph.com/issues/16157
* http://tracker.ceph.com/issues/16800
